src/module/migration/migrations/866-link-to-actor-size-again.ts

```diff
@@ -0,0 +1,30 @@
+import type { ActorPF2e } from "@actor";
+import type { ActorSourcePF2e } from "@actor/data/index.ts";
+import { isActorTypeSizeLinkable } from "@scene/helpers.ts";
+import { MigrationBase } from "../base.ts";
+
+export class Migration866LinkToActorSizeAgain extends MigrationBase {
+    static override version = 0.866;
+
+    override async updateActor(actorSource: ActorSourcePF2e): Promise<void> {
+        if (isActorTypeSizeLinkable(actorSource.type)) return;
+
+        if (actorSource.prototypeToken.flags.pf2e) {
+            actorSource.prototypeToken.flags.pf2e.linkToActorSize = false;
+            actorSource.prototypeToken.flags.pf2e.autoscale = false;
+        }
+    }
+
+    override async updateToken(tokenSource: foundry.documents.TokenSource, actor: ActorPF2e): Promise<void> {
+        if (!actor || !("type" in actor) || isActorTypeSizeLinkable(actor.type)) {
+            return;
+        }
+
+        mergeObject(tokenSource.flags, {
+            pf2e: {
+                linkToActorSize: false,
+                autoscale: false,
+            },
+        });
```

Comment:
This bypasses type checking of any kind

Reply:
Token sources don't have a type for pf2e flags, only the prep data does. There would be no type checking regardless unless we do that. Wasn't sure if its worth it for this but I'll try to figure that out I suppose.